spec/spec_helper.rb

```diff
@@ -93,15 +94,31 @@
   # Geocoding
   config.before(:each) { Spree::Address.any_instance.stub(:geocode).and_return([1,1]) }
 
+  default_country_id = Spree::Config[:default_country_id]
+  checkout_zone  = Spree::Config[:checkout_zone]
+  currency = Spree::Config[:currency]
   # Ensure we start with consistent config settings
-  config.before(:each) { Spree::Config.products_require_tax_category = false }
+  config.before(:each) do
```

Comment:
Is this how Spree does it as well? Can you add a reference?

Reply:
https://github.com/openfoodfoundation/spree/blob/step-6a/core/spec/spec_helper.rb#L59